index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link href="css/style.css" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Service page</title>
+  </head>
+  <body>
+    <div class="header_wrapper">
+      <header class="page_header flex_block">
+        <a href="javascript:;" class="link header_logo ">LO<span class="logo_span">GO</span></a>
+        <nav class="navigation header_navigation">
+          <ul class="list header_list navigation_list flex_block">
+            <li class="item"><a href="javascript:;" class="link navigation_link">Home</a></li>
```

Comment:
For future - add anchors for navigation. Also rewrite navigation without using list.